lib/board.rb

```diff
@@ -0,0 +1,122 @@
+require './lib/movable.rb'
+
+require './lib/evaluatable.rb'
+require './lib/player.rb'
+
+class Board
+  include Movable
+
+  include Evaluatable
+
+  def initialize
+    @player = Player.new
+    @board = [
+
+      %w[_ _ _],
+
+      %w[_ _ _],
+
+      %w[_ _ _]
+
+    ]
+
+    @converter = {
+
+      1 => [2, 0],
+
+      2 => [2, 1],
+
+      3 => [2, 2],
+
+      4 => [1, 0],
+
+      5 => [1, 1],
+
+      6 => [1, 2],
+
+      7 => [0, 0],
+
+      8 => [0, 1],
+
+      9 => [0, 2]
+
+    }
+  end
+
+  attr_accessor :board
+
+  attr_reader :player, :converter
+
+  def display
+    puts " #{board[0][0]} | #{board[0][1]} | #{board[0][2]}"
+
+    puts '-----------'
+
+    puts " #{board[1][0]} | #{board[1][1]} | #{board[1][2]}"
+
+    puts '-----------'
+
+    puts " #{board[2][0]} | #{board[2][1]} | #{board[2][2]}"
+  end
+
+  def play
+    result = ''
+
+    puts "\n\nTIC TAC TIE"
+
+    display
+
+    done = false
+
+    until done
+
+      puts "\nInsert Position"
+
+      number = gets.chomp.to_i
+
+      if board[converter[number][0]][converter[number][1]] == 'o' ||
+         board[converter[number][0]][converter[number][1]] == 'x'
+
+        puts 'This Position is Occupied Choice another one'
+        number = gets.chomp.to_i
```

Comment:
- [ ] Please remove puts and gets.chomp. 
![only_main_print](https://user-images.githubusercontent.com/58504655/97546857-d916e800-19ee-11eb-951d-40534b805f94.png)